Source/com/drew/metadata/exif/ExifTiffHandler.java

```diff
@@ -459,6 +460,12 @@ private boolean processMakernote(final int makernoteOffset,
             // http://exiv2.org/makernote.html
             pushDirectory(OlympusMakernoteDirectory.class);
             TiffReader.processIfd(this, reader, processedIfdOffsets, makernoteOffset + 12, makernoteOffset);
+        } else if ("OM SYSTEM\0\0\0II".equals(firstFourteenChars)) {
```

Comment:
The `II` suggests Intel byte ordering. Do we know if there's a similar `MM` variant for Motorola order?

Reply:
Not that I know of - this pattern matches all the photos I've taken with my OM1. I was under the assumption that the 'II' stood for the same indicator that the makernote was the Olympus second version since OM system bought the imaging division of Olympus and just stuck the new signature there.